redis/asyncio/cluster.py

```diff
@@ -1319,6 +1316,30 @@ async def close(self, attr: str = "nodes_cache") -> None:
             )
         )
 
+    def remap_host_port(self, host: str, port: int) -> Tuple[str, int]:
```

Comment:
related change?

Reply:
Necessary, if we are to use the `DelayProxy` for the cluster tests.  See `tests/test_asyncio/test_cwe_404.py`.  In its current form it is not actually doing anything.  The format is taken from the `redis-py-cluster` project, see PR description for more info.